src/cargo/ops/registry.rs

```diff
@@ -380,6 +380,19 @@ pub fn search(query: &str,
         };
         try!(config.shell().say(line, BLACK));
     }
+    if total_crates > limit as u32 && limit < 100 {
+        try!(config.shell().say(
+            format!("... and {} crates more (use --limit N to see more)",
+                    total_crates - limit as u32),
+            BLACK)
+        );
+    } else if total_crates > limit as u32 && limit >= 100 {
+        try!(config.shell().say(
+            format!("... and {} crates more (go to http://crates.io to see more)",
```

Comment:
I like a lot this idea, I have to find how to correctly build the url encoded q value. :smile: